index.html

```diff
@@ -0,0 +1,209 @@
+            <div class="awsome-icon">
+              <i class="fab fa-amazon"></i>
+            </div>
+            <h3 class="course-name">Amazon </h3>
+            </div>
+            <p class="text-description border">
+              A step-by-step guide for how to sell on Amazon, start an Amazon
+              FBA business & MASTER Amazon Seller Central!.from Your Home
+            </p>
+          </article>
+          <article class="course-card">
+            <div class="course-intro">
+            <div class="awsome-icon" id="lara">
+              <i class="fab fa-laravel"></i>
+            </div>
+            <h3 class="course-name">Laravel </h3>
+            </div>
+            <p class="text-description border">
+              PHP with Laravel for beginners - Become a Master in Laravel Learn
+              to master Laravel to make advanced applications like the real CMS
+            </p>
+          </article>
+          <article class="course-card">
+            
+            <div class="course-intro">
+
+              
+              <div class="awsome-icon">
+                <i class="fab fa-java"></i>
+            </div>
+            <h3 class="course-name">Java</h3>
+          </div>
+
+            <p class="text-description border">
+              If you don't know programming and want to start your career by
+              learning Java as your first language, you can start with this
+              course
+            </p>
+          </article>
+        </div>
+        <a href="#" class="courses-link"> See all courses</a>
+      </section>
+      <section class="our-team">
+        <h2 class="title-secondary">Top Content Developers</h2>
+        <div class="team-conatiner" id="team-container"></div>
+      </section>
+      <section class="partner-conatiner">
+        <h2 class="title-secondary">Trusted by</h2>
+        <div class="partners">
+          <div>
+            <img
+              src="images/c-1.png"
+              alt="partner logo"
+              class="partner-img"
+            />
+          </div>
+          <div>
+            <img
+              src="images/c-2.png"
+              alt="partner logo"
+              class="partner-img"
+            />
+          </div>
+          <div>
+            <img
+              src="images/c-3.png"
+              alt="partner logo"
+              class="partner-img"
+            />
+          </div>
+          <div>
+            <img
+              src="images/c-4.png"
+              alt="partner logo"
+              class="partner-img"
+            />
+          </div>
+     
+          <div>
+            <img
+              src="images/c-6.png"
+              alt="partner logo"
+              class="partner-img"
+            />
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer class="footer">
+      <div class="footer-logo">
+        <img src="images/logo.png" alt="">
+        <strong class="company-name">Web Evolution</strong>
```

Comment:
- [ ] This line of Html code should not be part of the codebase, kindly remove it as the text that you see in the provided design is part of the logo and not independent text.

Reply:
Resolved